nfts/scripts/deployAndMintNFT.js

```diff
@@ -0,0 +1,182 @@
+    },
+    to: {
+      email: process.env.USER_EMAIL_ADDRESS,
+    },
+    metadata: {
+      name: "Metakeep Tutorial NFT",
+      description: "Metakeep Tutorial NFT",
+      image: "https://metakeep.xyz/images/MetaKeep-1.png",
+      external_url: "https://metakeep.xyz",
+      attributes: [
+        {
+          trait_type: "Type",
+          value: "Tutorial",
+        },
+      ],
+    },
+  };
+
+  const options = {
+    method: "POST",
+    headers: headers,
+    body: JSON.stringify(requestBody),
+  };
+  console.log("NFT minting in process...");
+
+  const result = await fetch(url, options);
+
+  const resultJson = await result.json();
+
+  console.log("NFT minting response:");
+
+  console.log(resultJson);
+
+  if (!result.ok) {
+    // if the NFT minting transaction is not Queued, logs the error and exits the program.
+    console.log("NFT minting failed. HTTP Status Code: " + resultJson.status);
+    exit(1);
+  }
+
+  return resultJson;
+}
+
+async function main() {
+  env.config();
+
+  // check if the API key is set in the environment variables
+  checkAPIKey();
+
+  /* *************************************************************** Create Collection *************************************************************** */
+  console.log(
+    "***************************************************** Create Collection *****************************************************\n"
+  );
+  // Creates a New Collection
+  console.log("Creating a new collection...");
+  const resultJson = await createCollection();
+  console.log("Collection creation in progress...");
+
+  // Waits for the transaction to be mined
+  await waitUntilTransactionMined(resultJson);
+  console.log(
+    "NFT Collection created successfully! Collection Address: " +
+      resultJson.collection +
+      "\n"
+  );
+
+  // Waits for 5 seconds.
+  sleep(5000);
+
+  /* *************************************************************** Mint NFT *************************************************************** */
+  console.log(
+    "***************************************************** Mint NFT *****************************************************\n"
+  );
+
+  // Mints a new NFT
+  console.log("Minting a new NFT...");
+
+  const collection = resultJson.collection;
+
+  const mintResultJson = await mintNFT(collection);
+
+  console.log("NFT mint in progress....");
+
+  // Waits for the transaction to be mined
+
+  await waitUntilTransactionMined(mintResultJson);
+
+  console.log(
+    "NFT minted successfully! Transaction Hash: " +
+      mintResultJson.transactionHash +
+      "\n" +
+      "NFT Token Id:" +
+      mintResultJson.token +
+      "\n" +
+      "OpenSea Link (DEVELOPMENT): https://testnets.opensea.io/assets/mumbai/" +
+      collection +
+      "/" +
+      mintResultJson.token +
+      "\n" +
+      "OpenSea Link (PRODUCTION): https://opensea.io/assets/matic/" +
+      collection +
+      "/" +
+      mintResultJson.token +
+      "\n"
+  );
+}
```

Comment:
Why is this one big console.log?
Split this in multiple lines. `\n` will be automatically added

Reply:
done